lucene/core/src/java/org/apache/lucene/index/LiveIndexWriterConfig.java

```diff
@@ -480,6 +488,22 @@ public String getSoftDeletesField() {
     return softDeletesField;
   }
 
+  /**
+   * Expert: return the amount of time to wait for merges returned by by MergePolicy.findCommitMerges(...).
+   * If this time is reached, those merges will be aborted and we will wait again. If this time limit is reached again,
```

Comment:
This is no longer true?  We will just wrap up the commit w/ those merges that did finish, and let the still running merges finish to completion?

Reply:
Fixed!